README.md

```diff
@@ -113,20 +113,20 @@ follows subtly different paths which are still labeled as the same route. We
 parse the GTFS data into the following format:
 
  * `trip_id`:              A unique identifier for the trip. Taken from GTFS
- * `trip_headsign`:        The bus's public LED readout for the trip. Taken from 
+ * `trip_headsign`:        The bus's public LED readout for the trip. Taken from
```

Comment:
this is often not present. We'll want to make this an optional field if it is required elsewhere in code.

Reply:
Okay.